wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -30,6 +38,197 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     LockedWallet(nodeApi, chainQueryApi)
   }
 
+  /** We need a output so we can get proper tx size estimation */
+  private def getDummyDLCFundingOutputs(
+      amount: CurrencyUnit): Vector[TransactionOutput] = {
```

Comment:
output is fixed size regardless of `amount`, I don't think I follow why this is needed when you can directly compute the number of bytes or vbytes you need

Reply:
My thinking was that since we know the funding tx will have 3 outputs (1 dlc output, remote change, our change), that we try to fund a tx of that size. If we only try to fund a tx with 2 outputs (1 dlc output, 1 our change), we could end up in an edge case where we don't have enough fees to pay for all the different size

Reply:
still not sure how that ^ reasoning affects this method as my understanding is that the size returned by this method is constant right? Also we do sometimes have funding txs with only 1 or 2 outputs if the change ends up being below dust threshold